Index: virtuoso-opensource-6.1.6+dfsg/binsrc/samples/image_magick/Makefile.am
===================================================================
--- virtuoso-opensource-6.1.6+dfsg.orig/binsrc/samples/image_magick/Makefile.am
+++ virtuoso-opensource-6.1.6+dfsg/binsrc/samples/image_magick/Makefile.am
@@ -35,10 +35,10 @@ im_la_CFLAGS  += -I$(top_srcdir)/libsrc/
 im_la_CFLAGS  += -I$(top_srcdir)/libsrc/plugin 
 im_la_CFLAGS  += -I$(top_srcdir)/libsrc/langfunc 
 im_la_CFLAGS  += @ZLIB_INC@
-im_la_CFLAGS  += $(IM_CPPFLAGS)
+im_la_CFLAGS  += $(IM_CFLAGS)
 im_la_CFLAGS  += -D_GNU_SOURCE -DGLOBALREF=extern -D_USRDLL
 im_la_LDFLAGS = -module -avoid-version -export-dynamic -export-symbols im.exp -no-undefined
-im_la_LIBADD  = $(IM_LDFLAGS) $(IM_LIBS)
+im_la_LIBADD  = $(IM_LIBS)
 
 # ----------------------------------------------------------------------
 #
Index: virtuoso-opensource-6.1.6+dfsg/configure.in
===================================================================
--- virtuoso-opensource-6.1.6+dfsg.orig/configure.in
+++ virtuoso-opensource-6.1.6+dfsg/configure.in
@@ -1314,64 +1314,16 @@ AC_HELP_STRING([--disable-imagemagick],
 
 if test "x$with_im" != "xno"
 then
-    AC_PATH_PROG(IM_CONFIG, Wand-config, notfound, ["$with_im/bin:$PATH"])
-
-    if test "x$IM_CONFIG" = "xnotfound"
-    then
-       AC_MSG_WARN([The ImageMagick plugin will not be build])
-       with_im=no
-    fi
+    PKG_CHECK_MODULES(IM,MagickWand,[with_im="yes"],[dnl
+	with_im="no";dnl
+	AC_MSG_WARN([The ImageMagick plugin will not be build])dnl
+	])
 fi
 
-if test "x$with_im" != "xno"
-then
-    AC_MSG_CHECKING([ImageMagick library usability])
-
-    save_CPPFLAGS="$CPPFLAGS"
-    save_LIBS="$LIBS"
-    save_LD_LIBRARY_PATH="$LD_LIBRARY_PATH"
-
-    IM_CPPFLAGS=`"$IM_CONFIG" --cppflags`
-    IM_LDFLAGS=`"$IM_CONFIG" --ldflags`
-    IM_LIBS=`"$IM_CONFIG" --libs`
-
-    CPPFLAGS="$CPPFLAGS $IM_CPPFLAGS"
-    LIBS="$LIBS $IM_LDFLAGS $IM_LIBS"
-
-    LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$with_im/lib"
-    export LD_LIBRARY_PATH
-
-    AC_TRY_RUN([
-#include <stdio.h>
-#include <stdlib.h>
-#include <string.h>
-#include <time.h>
-#include <wand/magick-wand.h>
-MagickWand *magick_wand;
-int
-main ()
-{
-  MagickWandGenesis();
-  return 0;
-}],
-[
-  AC_MSG_RESULT(good)
-],
-[
-  AC_MSG_RESULT([bad. Check config.log for details])
-  with_im="no"
-  AC_MSG_WARN([The ImageMagick plugin will not be build])
-])
-
-    LIBS="$save_LIBS"
-    CPPFLAGS="$save_CPPFLAGS"
-    LD_LIBRARY_PATH="$save_LD_LIBRARY_PATH"
-fi
 
 AM_CONDITIONAL(WITH_IM, test "x$with_im" != "xno" )
-AC_SUBST(IM_CPPFLAGS)
-AC_SUBST(IM_LDFLAGS)
 AC_SUBST(IM_LIBS)
+AC_SUBST(IM_CFLAGS)
 
 
 ##########################################################################
