Description: Do not verify libssl version in configure.ac
Author: Andreas Tille <tille@debian.org>
Author: Andreas Beckmann <anbe@debian.org>
Forwarded: not-needed
Last-Update: 2024-03-06

--- a/configure.ac
+++ b/configure.ac
@@ -917,7 +917,7 @@ then
        #elif OPENSSL_VERSION_NUMBER < 0x30200000L
        /* OpenSSL version 3.1.x is supported */
 	#else
-	#error OpenSSL version too new
+	//#error OpenSSL version too new
 	#endif
 	    ],[
              AC_MSG_RESULT([$openssl_VERSION is supported])
